src/Collections-Strings/String.class.st

```diff
@@ -1857,6 +1883,14 @@ String >> isPatternVariable [
 	 ^self keywords anySatisfy: [:each | each first = $`]
 ]
 
+{ #category : #testing }
+String >> isPseudovariableName [
+	"Answer true if I am a pseudo-variable name.
+	#self isPseudovariableName -> true
+	"
+	^ self class pseudovariablesNames includes: self
+]
```

Comment:
Why is this in String? The only user is Slot. This should be moved to the Slot class...

```smalltalk
aSymbol isPseudovariableName ifTrue: [ 
		^ InvalidSlotName signalFor: aSymbol ].
```

```smalltalk
(self isPseudovariableName: aSymbol) ifTrue: [ 
		^ InvalidSlotName signalFor: aSymbol ].

```

I don't see the value of this being in `String`